Biohackathon 2025 - Project 9 (#3501)
galaxyproject/galaxy-hub: content/news/2025-12-17-EU_Biohackathon2025/index.md

- **ISCC support in Galaxy**
  - ISCC hash generator
  - Tools for hash verification and comparison in the making
- **Metadata & standards**
  - Extended **GeoJSON** support
  - Advanced **EDAM Bioimaging** maturity
    - Tool and tutorial annotation
    - Domain-specific term enrichment

Together, these improvements strengthen provenance, traceability, and FAIRness across Galaxy image analysis workflows.

## Work in progress (WIP)

Key efforts still evolving:

- **OMERO → Galaxy** integration for seamless image import  
- **ISCC tutorial** to support adoption and good practices  
- **Galaxy Imaging Lab** setup  
- **"Where to start" tutorial** for new users  
- **Community paper**
  - Authorship coordination
  - Outline and roadmap underway

➡️ Follow our progress on the [project board](https://github.com/orgs/FAIR-imaging/projects/4)

## What’s next

We’re aiming to:

- Write a report
- Finalize and stabilize all WIPs  
- Deliver **ready-to-use workflows** for image analysis in Galaxy  
- Publish a community paper including tutorials and training materials  
- Keep building an open, collaborative imaging community that makes image data science **FAIR**!

## Get involved

> Whether you're into Galaxy or any kind of imaging, your input is welcome. We’re always looking for feedback, tool testers, tutorial contributors, and creative ideas.

Reviewer: ```suggestion
Whether you're into Galaxy or any kind of imaging, your input is welcome. We're always looking for feedback, tool testers, tutorial contributors, and creative ideas.
```

If you're **passionate about open science and FAIR imaging**, join us, we're building this together.

### Communication Channels

Join our [Matrix channel](https://matrix.to/#/#galaxyproject_imaging:matrix.org) for live discussions and updates.

### Community Meetings

> We meet monthly via the **Euro-BioImaging Expert Group** on “FAIR Image Data Workflows”, chaired by Anne Fouilloux and Beatriz Serrano-Solano.

Reviewer: ```suggestion
We meet monthly via the **Euro-BioImaging Expert Group** on "FAIR Image Data Workflows", chaired by Anne Fouilloux and Beatriz Serrano-Solano.
```

## Why this project?

Image analysis is still fragmented between domains, infrastructures, repositories, and tools. Our goal? Simple but ambitious:  
Make image analysis more **interoperable, reusable, and scalable in Galaxy**, while staying **FAIR** from data to results.

## What we achieved

Concrete results from the hackathon:

- **BioImage Archive → Galaxy integration**
  - Added *BioStudies upload* support
  - Improved *BIA retrieval* tooling
- **ISCC support in Galaxy**
  - ISCC hash generator
  - Tools for hash verification and comparison
- **Metadata & standards**
  - Extended **GeoJSON** support
  - Advanced **EDAM Bioimaging** maturity
    - Tool and tutorial annotation
    - Domain-specific term enrichment

Together, these improvements strengthen provenance, traceability, and FAIRness across Galaxy imaging workflows.

## Work in progress (WIP)

Key efforts still evolving:

- **OMERO → Galaxy** integration for seamless image import  
- **ISCC tutorial** to support adoption and good practices  
- **Galaxy Imaging Lab** setup  

> - **“Where to start” tutorial** for new users  

Reviewer: See my comment below

```suggestion
- **"Where to start" tutorial** for new users  
```
